packages/api/api.go

```diff
@@ -865,6 +865,26 @@ func CallPAMAccess(httpClient *resty.Client, request PAMAccessRequest) (PAMAcces
 	return pamAccessResponse, nil
 }
 
+func CallPAMAccessApprovalRequest(httpClient *resty.Client, request PAMAccessApprovalRequest) (PAMAccessApprovalRequestResponse, error) {
+	var pamAccessApprovalRequestResponse PAMAccessApprovalRequestResponse
+	response, err := httpClient.
+		R().
+		SetResult(&pamAccessApprovalRequestResponse).
+		SetHeader("User-Agent", USER_AGENT).
+		SetBody(request).
+		Post(fmt.Sprintf("%v/v1/approval-policies/pam-access/requests", config.INFISICAL_URL))
+
+	if err != nil {
+		return PAMAccessApprovalRequestResponse{}, NewGenericRequestError(operationCallPAMAccess, err)
```

Comment:
is "operationCallPAMAccess" usage here intentional? it should be "operationCallPAMAccessApprovalRequest" right?